### DISCUSSION TITLE
Capturing raw images

### DISCUSSION THREAD

**Discussion by friedman101** [Stellarium/stellarium#1641] [Q&A] [answered]
Hello! I'm a new user of this awesome software. I'm curious if it's possible to capture a raw image as it would appear in the image sensor frame. Thanks!

**Comment by Zi7ar21** [comment, marked as answer]
If by raw you mean like a `*.dng`, `*.cr2`, or `*.nef` image file, then no. There isn't a point to saving a screenshot of a sky survey in a format that preserves camera sensor data, because the sky survey comes from a camera sensor that has already had the data it collected debayered and processed. However, I think tools to simulate using an observatory camera integrated into Stellarium would be quite fun. I have spent too much time messing around with the demo camera in programs like SharpCap 😂

(I hope I summed that up well)

**Reply by friedman101** [reply]
Thanks for the reply! 

Even if there is no way to save the image in a raw format, is it possible to save just the region which the sensor itself would see in a non-raw format (`png`, `jpg`, etc)?

**Reply by gzotti** [reply]
Make a screenshot. If you need to simulate the frame a sensor would deliver: Make a screenshot and trim on red box. Stellarium provides a simulation and nothing about processing actual camera data.

**Reply by friedman101** [reply]
Copy, thanks.
